scenarios/chapter1/01_The_Uprooting.cfg

```diff
@@ -302,32 +302,22 @@ This is the story of Kalenz, Landar, and of the elves in the first days of the h
 
         [message]
             id=Velon
-            message= _ "Lintanir's forces are barely strong enough to handle their dark hordes! We have no choice but to submit!"
+            message= _ "Until Elensiria learns of our plight, force of arms shalln't avail us! We have no choice but to submit!"
```

Comment:
What is Elensiria?

Reply:
It's the capital of the Lintanir elves seen in [Heir to the Throne, Scenario 21: Elven Council](https://github.com/wesnoth/wesnoth/blob/master/data/campaigns/Heir_To_The_Throne/scenarios/21_Elven_Council.cfg).